routing/base/astar_algorithm.hpp

```diff
@@ -47,7 +48,10 @@ class AStarAlgorithm
     return os;
   }
 
-  using TOnVisitedVertexCallback = std::function<void(TVertexType const &, TVertexType const &)>;
+  using OnVisitedVertexCallback = std::function<void(TVertexType const &, TVertexType const &)>;
+  // Callback used to path length from start/finish to edge including edge before adding edge to
```

Comment:
used to do what?
s/including edge/(including the edge itself)/

Reply:
fixed